index.html

```diff
@@ -4,263 +4,310 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
-  <!-- Material Icons -->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <title>Cookey-Portfolio-Setup</title>
+    <link
+    href="https://fonts.googleapis.com/css2?family=Andika&family=Inter:wght@300;400;500;600;700;800;900&family=Poppins:wght@200;300;500;600;700;800;900&family=Roboto:wght@100;300;400;500;700;900&family=Ubuntu:ital,wght@1,300&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
 </head>
 <body>
-  
-<div class = main_section>
  
-<header class="nav-bar">    
-  <a href=""><div class="logo-left">My Logo</div></a>
-  <nav><i class="material-icons">menu</i></nav>
-</header>
-
-<section class="headline">
-  <div class="container">
-   <h1 class="headline_text">I'm Cookey<br>Glad to see <br> you!</h1>
-    <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
-    </p>
- </div>
-  <h5>LET'S CONNECT</h5>
-
-  <div class="contacts">
-   <ul>
-    <li><a href=""><img src ="./images/twitter.svg" class="social-icons" alt="twitter">
-    </a></li>
-  
-  <li><a href=""><img src ="./images/linkedin.svg" class="social-icons" alt="twitter">
-  </a></li>
-
-  <li><a href=""><img src ="./images/medium.svg" class="social-icons" alt="twitter">
-  </a></li>
-  
-  <li><a href=""><img src="./images/github.svg" class="social-icons" alt="twitter">
-  </a></li>
-     
-  <li><a href=""><img src="./images/Vector.svg" class="social-icons" alt="twitter">
-  </a></li>
-   </ul>  
-  </div>
-</section>
-
-
-<section class="wrapper">
-  <div class="card1">
-      <div class="snapshot"><img src="./images/Snapshoot.svg" alt="tonic"></div>
-      <div class="block">
-          <h1 class="tonic_text">Tonic</h1>
-          <div class="subfram">
-              <p class="sub1">CANOPY</p>
-              <div class="circle"></div>
-              <p class="sub2">Back End Dev</p>
-              <div class="circle"></div>
-              <p class="sub2">2015</p>
-          </div>
+    <header>
+      <nav class="nav-bar">
+          <div class="logonav">
+            <a href="#">My Logo</a>
```

Comment:
- [ ]  You should add the margin similar to the one provided from Figma design. 
![image](https://user-images.githubusercontent.com/43877689/172959331-a231d0d2-2117-4ce2-b42c-bad297907f5d.png)